scientific_library/tvb/datatypes/projections.py

```diff
@@ -104,7 +107,7 @@ class ProjectionSurfaceEEG(ProjectionMatrix):
     Specific projection, from a CorticalSurface to EEG sensors.
     """
 
-    projection_type = Attr(field_type=str, default=EEG_POLYMORPHIC_IDENTITY)
+    projection_type = Attr(field_type=str, default=ProjectionsType.EEG.value)
```

Comment:
why not `field_type=ProjectionsType, default=ProjectionsType.EEG`? The actual value of an enum is a implementation detail.

or with neotraits support, perhaps more concisely,
```python
projection_type = EnumAttr(default=ProjectionsType.EEG)
```
with internally initializing `field_type=default.__class__`?

Reply:
Pushing this further the enum class could also allow generating instances from a enum member directly, since manually coding that is error-prone.